src/Symfony/Component/Console/Application.php

```diff
@@ -120,17 +119,10 @@ public function run(InputInterface $input = null, OutputInterface $output = null
         $this->configureIO($input, $output);
 
         try {
-            $e = null;
             $exitCode = $this->doRun($input, $output);
-        } catch (\Exception $x) {
-            $e = $x;
-        } catch (\Throwable $x) {
-            $e = new FatalThrowableError($x);
```

Comment:
this was effectively useless. below it was throwing the original exception ($x) in this case anyway, not $e. this could be removed in 2.7 as well. but there the debug component is still needed for the ConsoleExceptionEvent (instead of ConsoleErrorEvent). so the debug component dependency can only be removed in 4.0

Reply:
with 4.0 we can also get rid of handling both \Exception and \Throwable as it was already done in other places like `doRun`

Reply:
Created #24962 for 2.7